app/helpers/filter_routes_helper.rb

```diff
@@ -3,6 +3,10 @@ def announcements_filter_path(*objects)
     announcements_path(path_arguments(objects))
   end
 
+  def news_and_communications_filter_path(*objects)
```

Comment:
is this used anywhere?  I haven't found it yet

Reply:
weirdly, it doesn't seem to show up anywhere in the code base, but when removed proceeds to break the tests. Which is pretty odd?

Reply:
The joy of dynamic programming. It's created here: app/views/classifications/_document_list.html.erb

```
      see_more_url = public_send(
        "#{type}_filter_path",
        @classification,
        publication_filter_option: filter_option
      )
```